Resources/Textures/_Harmony/Clothing/Head/Hardsuits/security-warden.rsi/meta.json

```
@@ -0,0 +1,41 @@
+﻿{
+    "version": 1,
+    "license": "CC-BY-NC-SA-4.0",
+    "copyright": "Made by Palourde",
```

It's best to specify the exact username and where it's from so people can better reach out to you if needed
```suggestion
    "copyright": "Made by Palourde89 (GitHub)",
```